tests/src/test/scala/org/http4s/parser/UriParserSpec.scala

```diff
@@ -138,75 +138,11 @@ class UriParserSpec extends Http4sSpec {
       check(relativeUris)
     }
 
-    "parse absolute URI with fragment" in {
```

Comment:
are these tests not valid?

Reply:
There are from #4132 i see, but not why the tests where deleted.

Reply:
A request target can't have a fragment.  We're using the same model, so the types don't say we can't, but we're testing invalid behavior.

I am now questioning why the test on line 297 should be removed, though.

Reply:
That was my mistake, all these tests are nested under the `requestTarget` label so I deleted all of the ones that mentioned fragments. That test in particular uses `fromString` though